contrib/opentimelineio_contrib/application_plugins/rv/example_otio_reader/otio_reader.py

```diff
@@ -154,53 +167,92 @@ def _create_stack(in_stack, track_kind=None):
     return new_stack
 
 
-def _create_track(in_seq, _=None):
+def _create_track(in_seq, context=None):
+    context = context or {}
+
     new_seq = commands.newNode("RVSequenceGroup", str(in_seq.name or "track"))
     extra_commands.setUIName(new_seq, str(in_seq.name or "track"))
 
     items_to_serialize = otio.algorithms.track_with_expanded_transitions(
         in_seq
     )
 
-    track_kind = in_seq.kind
+    with set_context(context, track_kind=in_seq.kind):
+        new_inputs = []
+        for thing in items_to_serialize:
+            if isinstance(thing, tuple):
+                result = _create_transition(*thing, context=context)
+            elif thing.duration().value == 0:
+                continue
+            else:
+                result = create_rv_node_from_otio(thing, context)
 
-    new_inputs = []
-    for thing in items_to_serialize:
-        if isinstance(thing, tuple):
-            result = _create_transition(*thing, track_kind=track_kind)
-        elif thing.duration().value == 0:
-            continue
-        else:
-            result = create_rv_node_from_otio(thing, track_kind)
+            if result:
+                new_inputs.append(result)
 
-        if result:
-            new_inputs.append(result)
+        commands.setNodeInputs(new_seq, new_inputs)
+        _add_metadata_to_node(in_seq, new_seq)
 
-    commands.setNodeInputs(new_seq, new_inputs)
-    _add_metadata_to_node(in_seq, new_seq)
     return new_seq
 
 
-def _create_timeline(tl, _=None):
-    return create_rv_node_from_otio(tl.tracks)
+def _get_global_transform(tl):
+    # since there's no global scale in otio, use the first source with
+    # bounds as the global bounds
```

Comment:
I know we discussed adding an optional `global_bounds` similar to the `global_start_time` on the Timeline, but I wasn't sure where we landed on it.   If we had such a thing, it's exactly what I'd use here. I'm happy to add 
 & use`global_bounds` though if it makes sense in non-RV contexts as well -- should I do that?

Regardless though, I would still need this as a fallback in case we had clips with bounds but no `global_bounds`.

Reply:
makes sense!